### ISSUE TITLE
No such namespace warnings with ClojureScript 1.9.542

### ISSUE THREAD

**Issue by borkdude** [andrewmcveigh/cljs-time#96]
Same as #89 but for newer version of ClojureScript 1.9.542. Creating a new issue just to be sure. Failed with the following compiler settings:

```
  :optimizations :advanced
  :parallel-build true
  :pseudo-names true
  :pretty-print true
```

The errors I got were:

```
Compiling ClojureScript...
• public/js/app.js
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 567 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 567 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 568 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 568 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 569 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 569 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 570 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 570 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 571 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 571 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 592 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 592 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 593 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 593 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 594 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 594 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 595 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 595 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 596 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: Use of undeclared Var cljs-time/core at line 596 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/core.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 528 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/format.cljs
WARNING: Use of undeclared Var cljs-time/core at line 528 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/format.cljs
WARNING: No such namespace: cljs-time, could not locate cljs_time.cljs, cljs_time.cljc, or Closure namespace "" at line 532 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/format.cljs
WARNING: Use of undeclared Var cljs-time/core at line 532 /Users/Borkdude/.boot/cache/tmp/Users/Borkdude/dre/DataHub/app/1erw/-xybho3/public/js/app.out/cljs_time/format.cljs
```

**Comment by borkdude** [comment]
This is probably due some other dependency we are pulling in. I'll check. Closing for now.

**Comment by vikeri** [comment]
I'm also experiencing this issue.

**Comment by borkdude** [comment]
@vikeri Did you try from the new master? I've released a version here: https://clojars.org/org.clojars.borkdude/cljs-time/versions/0.5.0-SNAPSHOT
Also check your dependency tree to ensure cljs-time 0.4.0 isn't pulled in

**Comment by vikeri** [comment]
@borkdude That version worked! Thanks!

**Comment by andrewmcveigh** [comment]
OK, I'll try get a release out from master today!

**Comment by andrewmcveigh** [comment]
Released `0.5.0` to https://clojars.org/com.andrewmcveigh/cljs-time
